lua/nvim-treesitter-textobjects.lua

```diff
@@ -52,6 +52,7 @@ function M.init()
       lsp_interop = {
         module_path = "nvim-treesitter.textobjects.lsp_interop",
         enable = false,
+		border = "none",
```

Comment:
Can you have a closer look onto indentation here? It seems that tabs and spaces get mixed here

Reply:
Strange that it does that here. On my vim it is aligned with previous line. Will try changing my tabstop